web_widget_digitized_signature/static/src/xml/digital_sign.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates id="template" xml:space="preserve">
+    <t t-name="FieldSignature"  >
+        <div class = "cc">
+            <div class="signature oe_edit_only"/><br/>
+            <button class="oe_form_binary_file_clear oe_edit_only clear_sign"><span>Clear</span></button>
+            <span class="oe_edit_only save_sign"></span>
+        </div>
+    </t>
+    <t t-name="FieldBinaryImage-extend">
```

Comment:
this is not a very speaking name. Choose something that relates to the module name